playbook/app/pb_kits/playbook/pb_stat_value/stat_value.rb

```diff
@@ -4,10 +4,10 @@ module Playbook
   module PbStatValue
     class StatValue < Playbook::KitBase
       prop :unit
-      prop :value, type: Playbook::Props::Number
+      prop :value, type: Playbook::Props::Numeric
 
       def formatted_value
-        number_with_delimiter(value, delimiter: ",")
+        number_with_delimiter(value, delimiter: ",", separator: ".")
```

Comment:
This change isn't needed, but does no harm. Using a `.` as the separator for the decimal is an American standard.